source/cloud/azure/azure-vm-multi.md

```diff
@@ -12,6 +12,8 @@ Dask Cloud Provider can be installed via `conda` or `pip`. The Azure-specific ca
 $ pip install dask-cloudprovider[azure]
 ```
 
+Try running `pip install "dask-cloudprovider[azure]"` instead if you encounter a `zsh: no matches found` error.
```

Comment:
Instead of adding another line to the docs, I think it'd be simpler to just add single quotes to the code sample above. That should always work, in any shell. Could you please do that instead?